packages/app/src/runner/event-manager.ts

```diff
@@ -281,26 +281,49 @@ export class EventManager {
     this.reporterBus.on('studio:init:test', (testId) => {
       this.studioStore.setTestId(testId)
 
-      studioInit()
+      this.ws.emit('studio:init', ({ error }) => {
+        // eslint-disable-next-line no-console
+        console.error(error)
```

Comment:
Isn't this going to show a blank console error if error is undefined?

Reply:
Blurg. I caught a couple of those but not all. Updated